### DISCUSSION TITLE
Uppercase coding in BIRT.TYPE?

### DISCUSSION THREAD

**Discussion by hartenthaler** [FamilySearch/GEDCOM#135] [Q&A] [answered]
In https://gedcom.io/migrate/ there is written:
```
It is recommended that if AGE STILLBORN appears under any event in 5.5.1, then to maximize compatibility with various uses of stillbirth you should do all of the following when converting to 7.0:
1 Ensure there is a DEAT event with AGE 0y (unless a different DEAT.AGE was already present), and
2 Ensure there is a BIRT event with TYPE Stillborn (unless a different birth type was already present), and
3 Remove the original AGE STILLBORN line entirely, optionally adding a NOTE Stillborn to preserve the information that this event was tagged with that “age”.
```
Is it correct that in point 2 the coding "TYPE Stillborn" is used? Or should it be "TYPE STILLBORN"?

**Comment by tychonievich** [comment, marked as answer]
Because `BIRT`.`TYPE` has datatype `Text`, not `Enum`, the spec does not impose any limitations on its payload, including any case restrictions. The recommendation on the migration site is intended to make stillbirth relatively consistent in how it is represented despite this lack of spec-imposed limitation.

We have [a proposal](https://github.com/FamilySearch/GEDCOM/issues/106#issuecomment-1015872264) which I expect we'll implement in 7.1 to make stillbirth more consistently implemented. That issue also discusses some of the challenges of both 5.5.1 and 7.0's stillbirth representations.
